src/Innovt.CrossCutting.Log.Serilog/ActivityEnrich.cs

```diff
@@ -22,9 +22,11 @@ public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
 
         logEvent.AddOrUpdateProperty(new LogEventProperty("TraceId", new ScalarValue(activity.Id)));
```

Comment:
@welbert  Why do you want to add parent id?

Reply:
In AWS case, parent id represents the request id, so add to dont lose this information

Reply:
@michelmob